deploy/ansible/create-cluster.yml

```diff
@@ -24,7 +24,7 @@
     - name: Validate profile yaml-file
       validator:
         source: "{{ lookup('env', 'PROFILES_PATH') or '../profiles' }}/{{ profile }}.yml"
-        template: "{{ lookup('env', 'PROFILES_PATH') or '../profiles' }}/profiles_template.yaml"
+        template: "../profiles/profiles_template.yaml"
```

Comment:
I think that template should be parameterized as a source.

Reply:
for ADA and legion *PROFILES_PATH* has different values, due legion and ada profiles are located in different folders, BUT profile/secret templates should be single for any client and located in *profiles* folder in *legion* repo